src/components/components/club-info/club-info.js

```diff
@@ -25,14 +25,25 @@ import "./club-info.scss";
 export class ClubInfoItem extends React.PureComponent {
   render() {
     let classes = ["club-info__item"];
+    let link = this.props.link;
     if (this.props.major) classes.push("club-info__item--major");
-    if (this.props.map) classes.push("club-info__item--map");
+    if (this.props.map) {
+      classes.push("club-info__item--map");
+      if (this.props.lng && this.props.lat) {
+        link =
+          "https://www.openstreetmap.org/directions?from=&to=" +
```

Comment:
You should never modify a react prop, make a copy of the prop and use that one if you need to modify it